webapp/components/Comment/Comment.vue

```diff
@@ -54,6 +54,15 @@
         </button>
       </div>
       <ds-space margin-bottom="small" />
+      <base-button
+        :title="this.$t('post.comment.reply')"
+        icon="level-down"
+        class="reply-button"
+        circle
+        size="small"
+        v-scroll-to="'.editor'"
+        @click.prevent="reply"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T10:27:21Z" title="Thursday, January 16th 2020, 11:27:21 am +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -54,6 +54,14 @@
         </button>
       </div>
       <ds-space margin-bottom="small" />
+      <ds-button
+        :title="this.$t('post.comment.answer')"
+        icon="level-down"
+        @click.prevent="reply"
+        v-scroll-to="'.editor'"
+        class="answerbutton"
+        size="small"
+      ></ds-button>
```


please refactor to use `base-button`